js/src/HandDraw.ts

```diff
@@ -28,163 +27,155 @@ export class HandDraw extends Interaction {
 
     // Register the mouse callback when the mark view promises are
     // resolved.
-    this.set_lines_view().then(() => {
+    this.setLinesView().then(() => {
       this.d3el.on('mousedown', () => {
         return this.mousedown();
       });
-      this.set_limits();
+      this.setLimits();
     });
 
-    // Update line index
-    this.update_line_index();
-    this.listenTo(this.model, 'change:line_index', this.update_line_index);
-    this.model.on_some_change(['min_x', 'max_x'], this.set_limits, this);
+    this.model.on_some_change(['min_x', 'max_x'], this.setLimits, this);
   }
 
-  set_lines_view() {
+  private async setLinesView(): Promise<void> {
```

Comment:
methods names should be snake case?

Reply:
or is it overriding something else?

Reply:
Nope, I think we decided to follow the JupyterLab conventions, so no more snake_case :)

Reply:
oh!
and pretty is not complaining? so ok :)

Reply:
Some of them are still snake_case because I am waiting for us to work on super classes like `Mark` before renaming them